### PR TITLE
ci: add install all npm packages after cleanup

### LINKED ISSUE #90: Install all npm packages after cleanup

### PR DESCRIPTION
**Pull request by romanrostislavovich**
added next gulp commands:
- `install:npm` - installing all npm packages from `package.json`. Use only with another gulp task.

Closes #90

### CODE REVIEW

File: script/gulp/task.test.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ const testTrue = (test, schema, done) => {
         log.error(`FAIL: ${schema.description} ${test.description}`);
         log.error(`FILE: ${test.data.$ref}`);
         log.error(`MESSAGE: Validation failed`);
+        done();
```
Comment (xferra) [inline script/gulp/task.test.js]: It's unclear why you reverting this change after I've (probably) fixed it a few days ago. This way `done` will be never executed if the test passed, right?

File: script/gulp/task.install.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// External modules as aliases
+const gulp = core.amd.gulp;
+const config = core.cfg;
+const gulpInstall = core.amd.gulpInstall;
+
+// Automatically install npm packages with --save-dev flag
+const npm = () => gulp
+    .src(config.root.package)
+    .pipe(gulpInstall({
+        args: ['--save-dev'],
```
Comment (xferra) [inline script/gulp/task.install.js]: Why save dev?

File: gulpfile.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ gulp.task('lint', gulp.series('lint:all'));
  *
  * @task {clean}
  */
-gulp.task('clean', gulp.series('clean:all'));
+gulp.task('clean', gulp.series('clean:all', 'install:npm'));
```
Comment (xferra) [inline gulpfile.js]: So, if `clean:build` failed somehow - `install:npm` will not be executed, right? Then it's more logical to (just schematic idea):

```javascript
task('clean', 
parallel
[
series(clean-node, install) <--- reinstall actually,
]
clean-build
clean-release
etc.
)
```

Comment (xferra) [reply gulpfile.js]: BTW, as for me`install:npm` sounds like `we gonna install npm itself`, any better name?

**Review by xferra — CHANGES_REQUESTED** [review]
FYI

File: script/gulp/task.install.js (hunk 2 above)
Comment (romanrostislavovich) [reply script/gulp/task.install.js]: Because, all packages in ours `package.json` install as `dev`

File: script/gulp/task.test.js (hunk 1 above)
Comment (romanrostislavovich) [reply script/gulp/task.test.js]: was fixed

File: gulpfile.js (hunk 3 above)
Comment (romanrostislavovich) [reply gulpfile.js]: done

File: script/gulp/task.install.js (hunk 2 above)
Comment (xferra) [reply script/gulp/task.install.js]: This command expected to install packages listed in `package.json`. It should not install any new packages, so this arg is not needed.

**Review by xferra — COMMENTED** [review]
+1

**Comment by xferra** [comment]
@svoboda-rabstvo Please resolve conflicts.

**Comment by romanrostislavovich** [comment]
done

**Review by xferra — APPROVED** [review]
+1

**Pull request merged**